docs/_source/conf.py

```diff
@@ -20,18 +20,20 @@
 import sys
 
 # Get the project root dir, which is the parent dir of this
-project_path = os.path.join(os.path.dirname(__file__), '..' + os.path.sep + '..')
+project_path = os.path.join(
+    os.path.dirname(__file__), ".." + os.path.sep + ".."
```

Comment:
No real benefit breaking this away from a single line, I doubt it was breaking a lint?. If you want to "improve" this one way to that is to do something like:

```python
from os.path import join
from os.path import dirname


project_path = join(dirname(__file__), join("..", ".."))
```

Otherwise, the original was fine as is.